kotlinpoet/src/main/java/com/squareup/kotlinpoet/jvm/JvmAnnotations.kt

```diff
@@ -69,12 +69,24 @@ fun FunSpec.Builder.jvmName(name: String) = apply {
 }
 
 fun FunSpec.Builder.throws(vararg exceptionClasses: KClass<out Throwable>) =
-    throws(*(exceptionClasses.map(KClass<*>::asTypeName).toTypedArray()))
+    throws(exceptionClasses.toList())
 
 fun FunSpec.Builder.throws(vararg exceptionClasses: Type) =
-    throws(*(exceptionClasses.map(Type::asTypeName).toTypedArray()))
+    throws(exceptionClasses.toList())
 
-fun FunSpec.Builder.throws(vararg exceptionClasses: TypeName) = addAnnotation(
+fun FunSpec.Builder.throws(vararg exceptionClasses: TypeName) =
+    throws(exceptionClasses.toList())
+
+@JvmName("throwsKClasses")
```

Comment:
Let's remove all these cause they require name changes.

Reply:
Fixed in 4f12e7d. could you check?